src/Fhir.Anonymizer.Tool/Stu3-config-draft.json

```diff
@@ -0,0 +1,267 @@
+    {"path": "RequestGroup.action.action", "method": "redact"},
+    //{"path": "Communication.topic", "method": "redact"},
+    {"path": "Communication.payload", "method": "redact"},
+    {"path": "CommunicationRequest.payload", "method": "redact"},
+    {"path": "Coverage.subscriberId", "method": "redact"},
+    {"path": "Coverage.dependent", "method": "redact"},
+    //{"path": "Coverage.class.value", "method": "redact"},
+    //{"path": "Coverage.class.name", "method": "redact"},
+    {"path": "Coverage.network", "method": "redact"},
+    {"path": "Coverage.Group.group", "method": "redact"},  ///Newadd
+    {"path": "Coverage.Group.subGroup", "method": "redact"},   ///Newadd
+    {"path": "Coverage.Group.plan", "method": "redact"},   ///Newadd
+    {"path": "Coverage.Group.subPlan", "method": "redact"},   ///Newadd
+    {"path": "Coverage.Group.class", "method": "redact"},   ///Newadd
+    {"path": "Coverage.Group.subClass", "method": "redact"},   ///Newadd
+    {"path": "Coverage.Group.groupDisplay", "method": "redact"},  ///Newadd
+    {"path": "Coverage.Group.subGroupDisplay", "method": "redact"},   ///Newadd
+    {"path": "Coverage.Group.planDisplay", "method": "redact"},   ///Newadd
+    {"path": "Coverage.Group.subPlanDisplay", "method": "redact"},   ///Newadd
+    {"path": "Coverage.Group.classDisplay", "method": "redact"},   ///Newadd
+    {"path": "Coverage.Group.subClassDisplay", "method": "redact"},   ///Newadd
+    //{"path": "CoverageEligibilityRequest.insurance.businessArrangement", "method": "redact"},
+    //{"path": "CoverageEligibilityResponse.disposition", "method": "redact"},
+    //{"path": "CoverageEligibilityResponse.insurance.item.name", "method": "redact"},
+    //{"path": "CoverageEligibilityResponse.insurance.item.description", "method": "redact"},
+    //{"path": "CoverageEligibilityResponse.insurance.item.benefit.allowed", "method": "redact"},
+    //{"path": "CoverageEligibilityResponse.insurance.item.benefit.used", "method": "redact"},
+    //{"path": "CoverageEligibilityResponse.preAuthRef", "method": "redact"},
+    {"path": "EligibilityRequest.businessArrangement", "method": "redact"}, ///Newadd
+    {"path": "EligibilityResponse.disposition", "method": "redact"},   ///Newadd
+    {"path": "EligibilityResponse.insurance.benefitBalance.name", "method": "redact"},   ///Newadd
+    {"path": "EligibilityResponse.insurance.benefitBalance.description", "method": "redact"},   ///Newadd 
+    {"path": "EligibilityResponse.insurance.benefitBalance.financial.allowed", "method": "redact"},   ///Newadd
+    {"path": "EnrollmentResponse.disposition", "method": "redact"},
+    {"path": "Claim.supportingInfo.value", "method": "redact"},
+    {"path": "Claim.insurance.businessArrangement", "method": "redact"},
+    {"path": "Claim.insurance.preAuthRef", "method": "redact"},
+    {"path": "ClaimResponse.disposition", "method": "redact"},
+    {"path": "ClaimResponse.insurance.preAuthRef", "method": "redact"}, //Modified

```

Comment:
Do we need to redact 'ClaimResponse.insurance.businessArrangement' ?

Reply:
It is free text

Reply:
Yes. I mean this path is missing..

Reply:
The path is here 
![image](https://user-images.githubusercontent.com/57981365/86438170-9d1d9000-bd38-11ea-91ee-765cefb39317.png)

Reply:
Never mind.